README.md

```diff
@@ -1,14 +1,25 @@
 # Firebase Auth with Google and React Context API   
+Quick links 
+[Set Up](#project-set-up)
+[Firebase API](firebase-api)
+[React useContext()](react-usecontext()) 
+[UserContext](managing-usercontext)
+[Auth State](auth-and-state)
+[Summary](#putting-it-all-together)
 
 This project will walk you through the steps to set up User Authentiaction via Google sign in using Firebase, React and React Context.  
```

Comment:
What is authentication? Why do we use it? How do we use it? 

Just add a couple sentences to 'set the scene' about why we are learning this.

If you have screenshots of your app, (and have time to do so, please include a few throughout) - it will help students understand what success looks like/ what they are building. I know it is time consuming to add screenshots, so don't worry if you are out of time.